src/uu/uniq/src/uniq.rs

```diff
@@ -247,6 +247,23 @@ fn opt_parsed<T: FromStr>(opt_name: &str, matches: &ArgMatches) -> UResult<Optio
     })
 }
 
+fn obsolete_skip_field(matches: &ArgMatches) -> Option<usize> {
+    for i in 0..=9 {
+        let mut s: String = i.to_string();
+        let v = matches.get_one::<String>(&s);
+
+        if matches.contains_id(s.as_str()) {
+            if let Some(k) = v {
+                s.push_str(k);
+                return s.parse::<usize>().ok().or(Some(i));
```

Comment:
This is not great, because any error is silently ignored. For example:
```
uniq -10b
```
is just run as
```
uniq
```
We should probably show an error in that case.